docker/mosaiq/docker-compose.yml

```diff
@@ -16,9 +16,9 @@ version: "3.6"
 
 services:
   mosaiq-mssql:
-    image: mcr.microsoft.com/mssql/server:2017-latest
```

Comment:
I needed to make this change so that the docker file worked on an Ubuntu instance installed within a VM on an aarch64 Mac